GUI/Util/GlobalGuiSettings.cs

```diff
@@ -112,19 +116,46 @@ public static bool CheckTopNPeaks(string text)
             return true;
         }
 
-        public static bool CheckMinRatio(string text)
+        public static bool CheckMinimumAllowedIntensityRatio(string text)
         {
-            if (!double.TryParse(text, NumberStyles.Any, CultureInfo.InvariantCulture, out double minRatio) || minRatio < 0 || minRatio > 1)
+            if (!double.TryParse(text, NumberStyles.Any, CultureInfo.InvariantCulture, out double minRatio) || minRatio < 0 || minRatio > 1 || text == null)
             {
                 MessageBox.Show("The minimum intensity ratio must be between zero and one. \n You entered " + '"' + text + '"');
                 return false;
             }
             return true;
         }
 
-        public static bool CheckPrecursorMassTolerance(string text)
+        public static bool CheckWindowWidthThomsons(string text)
         {
+            if (double.TryParse(text, NumberStyles.Any, CultureInfo.InvariantCulture, out double windowWidthThomsons))
```

Comment:
refactor with "!"to make this an if-else, instead of the current if-else if-else

Reply:
done